FINALMENTE, MILANESAS V2

TotoMC-13 opened the pull request:
## About The Pull Request
Agrega milanesas al juego
![image](https://user-images.githubusercontent.com/52181149/93400964-6810dc00-f857-11ea-94f8-5e514d283dfe.png)
![image](https://user-images.githubusercontent.com/52181149/93404987-22a5dc00-f862-11ea-9702-60be72f4807e.png)

## Why It's Good For The Game
Trae otro alimento al juego para tener mas variedad, tambien incluye pan rallado para poder hacer las milanesas y un rallador de queso para poder tener pan rallado
## Changelog
:cl:
add: Milanesa cruda, milanesa, rallador de queso y pan rallado
imageadd: raw_milanesa, milanesa, cheese_grater y bread_crumbs
/:cl:

TotoMC-13 committed:
FINALMENTE, MILANESAS

SON MILANESAS QUE MAS QUERES QUE TE DIGA

Silverexperience commented on code/hispania/modules/food/food.dm:
```diff
@@ -19,3 +19,34 @@
+	icon = 'icons/hispania/obj/food/food.dmi'
+	icon_state = "pan_rallado"
+	filling_color = "#EC3924"
+	cooked = TRUE
```
Pon un espacio al final del código.

Silverexperience commented on code/hispania/modules/food/recipes_microwave.dm:
```diff
@@ -0,0 +1,6 @@
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/milanesa_cruda
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/milanesa
```
Aqui tambien.

Silverexperience reviewed (changes requested):
Los sprites son muy raros, no parecen unas milanesas, además no me suena que los sprites de cev eris tengan contornos.

TotoMC-13 committed:
Milanesas V2

## About The Pull Request
Agrega milanesas al juego
## Why It's Good For The Game
Trae otro alimento al juego para tener mas variedad
## Changelog
:cl:
add: Milanesa cruda, milanesa, rallador de queso y pan rallado
imageadd:

/:cl:

TotoMC-13 committed:
YA ESTA

TotoMC-13 committed:
Ya

TotoMC-13 committed:
YA v238

TotoMC-13 committed:
Correcciones

TotoMC-13 committed:
Soy tonto
